lineman/app/models/user_records_interface.coffee

```diff
@@ -3,4 +3,13 @@ angular.module('loomioApp').factory 'UserRecordsInterface', (BaseRecordsInterfac
     model: UserModel
 
     updateProfile: (user) ->
-      @restfulClient.post 'update_profile', user.serialize()
\ No newline at end of file
+      @restfulClient.post 'update_profile', user.serialize()
```

Comment:
It's starting to look like it should subclass the restfulClient to me. What do you think?